yang/ietf-ntw-energy-saving.yang

```diff
@@ -52,7 +59,103 @@ module ietf-ntw-energy-saving {
     reference
       "RFC XXXX: YANG Data Models for Energy Saving Management";
   }
+  /*
+  Identity
+  */
+  identity energy-saving-power-state {
+    description
+      "Base identity for power state.";
+    reference
+      "RFC 7326: Energy Management Framework";
+  }
+
+  identity off-state {
```

Comment:
I would delete this as this is part of the common model

Reply:
OK